### ISSUE TITLE
invert chrome:// pages like bookmarks

### ISSUE THREAD

**Issue by yamlCase** [alexanderby/darkreader#120]
It would be good to also have the chrome:// pages invertable.  Right now when I go to chrome://bookmarks Dark Reader has the "Toggle" feature grayed out.

**Comment by nemanjan00** [comment]
I do not think this is possible since Chrome does not allow plugins to run any code on those pages.

**Comment by alexanderby** [comment]
You are right, this pages are protected from extensions.
